config/routes.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'data_services#index'
```

Comment:
Can you move this at the bottom? Routes work as cascades, let's have the root as the last entry